src/cleed++/Model.hh

```
@@ -0,0 +1,44 @@
+ *
+ * @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+ *
+ *********************************************************************/
+
+/*! \file
+ *
+ * Provides a base class for CLEED atomic models.
+ */
+
+#ifndef __MODEL_HH__
+#define __MODEL_HH__
```

Clang (14.0.6)

[warning] declaration uses identifier '__MODEL_HH__', which is a reserved identifier

(bugprone-reserved-identifier)